bin/revert

```diff
@@ -41,7 +41,11 @@ def post_comment(
         revert_sha: str,
         headers: dict[str, str],
 ) -> None:
-    data = {'body': f'PR reverted: {revert_sha}'}
+    data = {
+        'body': f'PR reverted: {
+            revert_sha
+        }.\n\nDid you leave a comment describing why this PR was reverted?',
```

Comment:
this syntax only works in python 3.12 which we can't guarantee is available

Reply:
Yeah, this is what flake8 did when i ran pre-commit locally. Let me see why

Reply:
or maybe autopep8, actually not sure which one's actually doing the formatting